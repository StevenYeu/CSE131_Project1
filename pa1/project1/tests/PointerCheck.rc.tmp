Error, "project1/tests/PointerCheck.rc": 
  Referenced field y not found in type MY.
Compile: failure.
