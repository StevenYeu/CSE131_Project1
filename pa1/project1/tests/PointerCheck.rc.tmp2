  Referenced field y not found in type MY.
Compile: failure.
