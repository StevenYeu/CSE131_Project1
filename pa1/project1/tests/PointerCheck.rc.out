Error, "/tmp/cs131_testrunner_BNarMF/file.rc": 
  Referenced field y not found in type MY.
Compile: failure.
